xml/http_servers.xml

```diff
@@ -2380,6 +2475,18 @@
     <param pos="0" name="service.vendor" value="nginx"/>
     <param pos="0" name="service.cpe23" value="cpe:/a:nginx:nginx:-"/>
   </fingerprint>
+  <fingerprint pattern="^nginx\/?(:?[\d.]+)?">
+    <description>nginx with version info and/or mods</description>
+    <example service.version="0.8.53">nginx/0.8.53 + Phusion Passenger 3.0.0 (mod_rails/mod_rack)</example>
+    <example>nginx/0.8.53</example>
+    <example>nginx + Phusion Passenger 5.1.11</example>
```

Comment:
Strange.  So we get an empty string for the version here?  May be worth two fingerprints, one with a version, one without.

Reply:
We have quite a few other fingerprints that can have an empty capture if the value isn't there.  If you like I can split this one up.